packages/block-editor/src/components/block-card/index.js

```diff
@@ -8,7 +8,7 @@ import deprecated from '@wordpress/deprecated';
  */
 import BlockIcon from '../block-icon';
 
-function BlockCard( { title, icon, description, blockType } ) {
+function BlockCard( { title, icon, description, blockType, backButton } ) {
```

Comment:
Removing this unused back button at https://github.com/WordPress/gutenberg/pull/43806